truss-chains/truss_chains/example_chainlet.py

```diff
@@ -1,31 +1,35 @@
+import random
+
+# for more on chains, check out https://truss.baseten.co/chains/intro
 import truss_chains as chains
 
 
-class DummyGenerateData(chains.ChainletBase):
-    def run_remote(self) -> str:
-        return "abc"
+# By inhereting chains.ChainletBase, chains will know to create a chainlet that hosts the RandInt class
```

Comment:
"chains will": leads to overloading the term "chains" with a instance of a chain and chains as a framework. You could disambiguate this by writing "the framework will know" or "the chains framework will know".

Reply:
"chains framework" sounds great - ty